### ISSUE TITLE
Trending packages

### ISSUE THREAD

**Issue by davecyen** [meteor/madewith2#8]
Make it easier for community to discover new packages that are on the rise

**Comment by Slava** [comment]
This is implemented in the new atmosphere (http://beta.atmospherejs.com/), does MadeWith really need trending packages?

**Comment by jadeqwang** [comment]
I agree that Atmosphere has it. The reason why MadeWith has package tracking (and the feature I'll add at some point) is: for a given package, which apps are using it, and can I look at their source code as an example of that usage.

**Comment by jadeqwang** [comment]
But having a "package" view in MW is probably not a bad idea. I'll mull over what that would look like.
